schedule/templates/index.html

```diff
@@ -0,0 +1,35 @@
+{% extends "master.html" %}
+
+{% block main_content %}
+
+<div class="col-xs-6" id="user-toolbar">
+    {% if user.image %}
+        <a href="#"><img height="30" class="img-circle"
+                         src="/media/{{ user.image }}"></a>
```

Comment:
Check how to use static urls in templates - it should be something like {% static user.image %} instead of hardcoding /media/. 

Imagine that you will need to change /media/ path to another one - it would be painful with hardcoding.